crates/ruff_workspace/src/configuration.rs

```diff
@@ -185,7 +185,7 @@ impl Configuration {
             exclude: FilePatternSet::try_from_iter(format.exclude.unwrap_or_default())?,
             extension: self.extension.clone().unwrap_or_default(),
             preview: format_preview,
-            unresolved_target_version: target_version,
+            unresolved_target_version: target_version.unwrap_or_default(),
```

Comment:
I thikn I'd prefer to instead leave line 167 unchanged to only have a single `unwrap_or_default` in `into_settings`.